src/domain/event/eventCard/__tests__/EventCard.test.tsx

```diff
@@ -57,15 +57,16 @@ test('should go to event page by clicking event card', () => {
   expect(history.location.pathname).toEqual('/');
 
   userEvent.click(
-    screen.queryByRole('link', {
+    screen.getByRole('link', {
       name: translations.event.eventCard.ariaLabelLink.replace(
         '{{name}}',
-        event.name.fi
+        // eslint-disable-next-line @typescript-eslint/no-explicit-any
+        event.name.fi as any
       ),
     })
   );
 
-  expect(history.location.pathname).toEqual('/fi/event/123');
```

Comment:
as the backward compability was left, we could check here both `event/` and `events/` ?

Reply:
yeah maybe, but that is a totally different test in a different place?

Reply:
I read this very badly, it's about the history here. I thought that we just followed the route. But there could be a test in AppRoute or somewhere. No need to do that if it has too much work.